src/Compatibility/Core/src/Compatibility-net6.csproj

```diff
@@ -8,6 +8,7 @@
     <iOSRoot>iOS\</iOSRoot>
     <WindowsRoot>Windows\</WindowsRoot>
     <IsPackable>false</IsPackable>
+    <EnableDefaultEmbeddedResourceItems>false</EnableDefaultEmbeddedResourceItems>
```

Comment:
Should you maybe check `$(BuildingInsideVisualStudio)` is `true`? Is it only broken inside VS?

Reply:
nop from cli using msbuild is the same

Reply:
So maybe could check if `$(MSBuildRuntimeType)` is `Full`?